addons/mail/static/tests/thread/thread.test.js

```diff
@@ -256,8 +257,8 @@ test("mention a channel with space in the name", async () => {
     await openDiscuss(channelId);
     await insertText(".o-mail-Composer-input", "#");
     await click(".o-mail-Composer-suggestion");
-    await contains(".o-mail-Composer-input", { value: "#General good boy " });
-    await press("Enter");
+    await contains(".o-mail-Composer-input", { text: "#General good boy" });
+    await click("button[title='Send']");
```

Comment:
Same as previous comment, and won't comment for other occurrences below in the diff:
Why replace the `press Enter` to `click Send`? Press Enter is used 99.9% of the time in desktop and this statement is much shorter too (= test more readable).